src/modules/home/Home.js

```diff
@@ -147,6 +147,35 @@ const Home = (props) => {
     };
   }, [props.currentImageBasemap, customMaps]);
 
+  useEffect(() => {
+    if (props.isImageModalVisible) {
+      populateData();
+    }
+    else imageSlideShowData = [];
+  }, [props.isImageModalVisible]);
+
+  const populateData = () => {
```

Comment:
populateData is kind of a broad name, especially for a function in Home. This doesn't really tell me what it is doing.